CHANGELOG.md

```diff
@@ -5,6 +5,10 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/)
 and this project adheres to [Semantic Versioning](http://semver.org/).
 ## Unreleased
 
+### UNRELEASED
```

Comment:
It seems `unreleased` section was already there.

Reply:
I wouldn't worry too much about this, we'll clean the changelog up when we kick out the release (which I'll kick one out this week sometime).